package.json

```diff
@@ -101,6 +101,10 @@
     },
     "scripts": {
         "clean": "rm -rf ./out",
+        "vscode:prepublish": "webpack --mode production",
+        "webpack": "webpack --mode development",
+        "webpack-dev": "webpack --mode development --watch",
+        "test-compile": "tsc -p ./",
```

Comment:
Is this something invoked by webpack? I see it's in the guide but is that only to showcase testing and the launch configuration?

Reply:
I'm not sure if/where this command would be called.  It's also very similar to the `compile`